packages/js/src/utils/errors.ts

```diff
@@ -16,6 +16,9 @@ const logtoErrorCodes = Object.freeze({
     verification: {
       invalidIat: 'Invalid issued at time',
     },
+    decode: {
```

Comment:
why not just idToken.invalidToken ?

Reply:
Since it's used in `decodeIdToken`.

Add `decode` keyword for describing this error code is used in the `decode` scenario.

Reply:
Just like the error code `idToken.verification.invalidIat` above.

Reply:
@simeng-li Improved, PTAL.